tests/helpers.py

```diff
@@ -0,0 +1,389 @@
+        if self.prevref is None or self.prevref.startswith(('ftp', 'http')):
+            os.environ[self.refstr] = p + os.sep
+            self.use_ftp_crds = True
+
+        # This controls astropy.io.fits timeout
+        conf.remote_timeout = self.timeout
+
+    def teardown_class(self):
+        """Reset path and variables."""
+        conf.reset('remote_timeout')
+        os.chdir(self.prevdir)
+        if self.use_ftp_crds and self.prevref is not None:
+            os.environ[self.refstr] = self.prevref
+
+    def get_input_file(self, filename, refsep='$'):
+        """
+        Download or copy input file (e.g., RAW) into the working directory.
+        The associated CRDS reference files in ``refstr`` are also
+        downloaded, if necessary.
+        """
+        download_file_cgi(self.tree, self.input_loc, filename,
+                          timeout=self.timeout)
+        ref_files = ref_from_image(filename)
+
+        for ref_file in ref_files:
+            if refsep not in ref_file:  # Local file
+                download_file_cgi(self.tree, self.input_loc, ref_file,
+                                  timeout=self.timeout)
+            else:  # Download from FTP, if applicable
+                s = ref_file.split(refsep)
+                refdir = s[0]
+                refname = s[1]
+                if self.use_ftp_crds:
+                    download_crds(refdir, refname, timeout=self.timeout)
+
+    def compare_outputs(self, outputs, raise_error=True):
+        """
+        Compare CALXXX output with "truth" using ``fitsdiff``.
+
+        Parameters
+        ----------
+        outputs : list of tuple
+            A list of tuples, each containing filename (without path)
+            of CALXXX output and truth, in that order.
+
+        raise_error : bool
+            Raise ``AssertionError`` if difference is found.
+
+        Returns
+        -------
+        report : str
+            Report from ``fitsdiff``.
+            This is part of error message if ``raise_error=True``.
+
+        """
+        all_okay = True
+        creature_report = ''
+
+        for actual, desired in outputs:
+            # Get "truth" image
+            s = download_file_cgi(self.tree, self.ref_loc, desired,
+                                  allow_remote_ref=True, timeout=self.timeout)
+            if s is not None:
+                desired = s
+
+            fdiff = FITSDiff(actual, desired, rtol=self.rtol, atol=self.atol,
+                             ignore_keywords=self.ignore_keywords)
+            creature_report += fdiff.report()
+
+            if not fdiff.identical and all_okay:
+                all_okay = False
+
+        if not all_okay and raise_error:
+            raise AssertionError(os.linesep + creature_report)
+
+        return creature_report
+
+
+@use_calacs
+class BaseACS(BaseCal):
+    refstr = 'jref'
+    prevref = os.environ.get(refstr)
+    input_loc = 'hstcal/acs/calacs_e'
```

Comment:
@mdlpstsci , in the current state of the POC, input files are assumed to be in a pre-defined location within these sub-folder, so to change the input data, you need to copy the new versions here.

Just below this line is where the "truth" folder is defined. These are following pandokia file system layout but we certainly don't have to do it this way going forward.

Does this answer you question about how to change the data files for a test?